joomla/joomla-cms
# com_installer discover view: searchtools ordering and some improvements

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_installer discover view.

Also:
- now discovers the extensions by default onload (if not discovered before).
- the columns were reordered to match the "Manage" view.
- corrects bug where the folder filter has not selecting the search button, and other small graphical issues.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/13027673/ba405dbc-d24f-11e5-8f91-48072f726efa.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/13027666/6c20f862-d24f-11e5-878f-2f5864a87377.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to the database "prefix_extensions" table and delete the "beez3" and "hathor" rows (or other if you want).
3. Go to Extensions -> Manage -> Discover (you will see hathor and beez3 there)
4. Check the search, ordering select box and the filter are working fine.
5. Test desktop, tablet and mobile view (ex: reduce browser width).
6. Install hathor and beez here and check if ok.
